CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.4
+* Update database mappings as follows:
+ - Use span name as remote dependency `name`
+ - Use span database statement as remote dependency `data`. If database statement is not present use `operation name`.
+ - If target derived from `peerService` or `net*` attributes is not null, then append span database name (if present) using ` | ` delimiter to form remote dependency target.    
```

Comment:
```suggestion
* Database dependency names are now more concise, e.g. `SELECT mydatabase.mytable` instead of `SELECT x, y, z, ... from ... where ...........` (the full sanitized query is still captured in the dependency data field)
* Database dependency target field slightly updated to enable better U/X integration
```